inst/include/common/information.hpp

```diff
@@ -479,6 +442,11 @@ class Information {
                << std::endl;
     }
     INFO_LOG << "Completed initialization of all populations." << std::endl;
+
+    //setup priors, random effect, and data density components
+    INFO_LOG << "Setup priors." << std::endl;
```

Comment:
The comment on line 446 mentions setting up three things, do we need to add INFO_LOG for setting up random effects and data here as well?

Reply:
I removed the INFO_LOG comment.